packages/protobuf/src/messages-schema.ts

```diff
@@ -2175,7 +2175,12 @@ export const EthereumTypedDataSignature = Type.Object(
 );
 
 export type EvoluGetNode = Static<typeof EvoluGetNode>;
-export const EvoluGetNode = Type.Object({}, { $id: 'EvoluGetNode' });
+export const EvoluGetNode = Type.Object(
+    {
+        proof_of_delegated_identity: Type.String(),
```

Comment:
@marekrjpolak this type is likeyl wrong. When I send here string that protobuf filled it, in throws and array. It expects `Buffer / ArrayBufferLike`.

So the type in FW repo is wrong, or?